app/views/councillor_contributions/new.html.haml

```diff
@@ -0,0 +1,17 @@
+%h1 Add a new councillor for #{@authority.full_name}
+%h2 The councillor(s) you submitted:
+= form_for [@authority, @councillor_contribution], url: authority_councillor_contributions_path do |f|
+  %ul
+    - @councillor_contribution.suggested_councillors[0...-1].each do |s|
+      %li
+        %dt Name
```

Comment:
These `%dt` and `%dd` elements are currently missing their `%dl` parent, see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt